flutter-studio/src/io/flutter/profiler/FlutterAllMemoryData.java

```diff
@@ -24,7 +24,7 @@
 import java.util.concurrent.TimeUnit;
 import java.util.function.Function;
 
-// Refactored from Android 3.3 adt-ui code.
+// Refactored from Android 3.2 adt-ui code.
```

Comment:
Isn't Android on version 9 now? I think these "Android 3.2" should be "Android Studio 3.2" everywhere.

But you can do it later.

LGTM